src/test/java/com/commercetools/sync/commons/utils/CustomUpdateActionUtilsTest.java

```diff
@@ -546,6 +546,22 @@ void buildNewOrModifiedCustomFieldsUpdateActions_WithNoNewOrModifiedCustomFields
         assertThat(updateActions).isEmpty();
     }
 
+    @Test
+    void buildNewOrModifiedCustomFieldsUpdateActions_WithNullCustomFields_ShouldNotBuildActions() {
```

Comment:
nice that you added this unit test. Although, I don't like that we test these protected methods. But it was a problem that was already there. I guess we should make these utils private and test them indirectly by testing the caller method. But this is something that we should not take care about right now I guess.

I will create an issue for it.

Reply:
https://github.com/commercetools/commercetools-sync-java/issues/434